frontend/src/components/common/wrappers/ModalWrapper.js

```diff
@@ -86,11 +86,12 @@ export class ModalWrapper extends Component {
             zIndex: 100,
           }}
         />
-        {ChildComponent && (
-          <ChildComponent
+        {this.props.content && (
+          <AddPartyComponentWrapper
             closeModal={this.props.closeModal}
```

Comment:
It seems like only a small subset of models deal with parties, and thus need to be concerned with `AddPartyComponentWrapper`, what's the purpose of wrapping every modal in this check?

Reply:
Maybe its the naming that's throwing me off, this is more so a CarouselWrapper? Where the second form could be anything, but in this situation/point in time it's an `AddPartyFrom`?

Reply:
Good thing to notice :) Primarily, I thought the usability trade-off was worth putting it here. Any form can use PartySelectField and have its behavior available without additional code. Another pro is that nobody has to examine other uses of it in the code base and selectively copy what they determine to be the pieces of code required in different files, and that there is less code duplication in the code-base. Parties are pretty core to the application and the assumption is that a reasonable number of new forms will include dealing with parties.

Reply:
Currently the second form is AddPartyForm only, and some of the logic is specific to adding parties, but if we have need for that to be generic it could probably be modified to be generic.

Reply:
ah okay, Thanks for the explanation, I pulled down the branch and played around with it, it's a very smooth transition.

Could you add a comment to briefly explain what's happening in this file? I can see how this would be confusing for future devs.

Reply:
For sure. I'll go though and add some extra comments. 👍

Reply:
Added extra comments where I thought things might not be intuitive.